scripts/error-codes/__tests__/dev-expression-with-codes-test.js

```diff
@@ -25,18 +25,17 @@ function compare(input, output) {
 
 var oldEnv;
 
-describe('dev-expression', function() {
```

Comment:
This removes the describe block and unindent everything. Sadly github doesn't have support for hiding whitespace-only changes :(